docs/apache-airflow/concepts/scheduler.rst

```diff
@@ -60,6 +60,34 @@ In the UI, it appears as if Airflow is running your tasks a day **late**
 
     You should refer to :doc:`/dag-run` for details on scheduling a DAG.
 
+DAG File Processing
+-------------------
+
+The Airflow Scheduler is responsible for turning the Python files contained in the DAGs folder into DAG objects that contain tasks to be scheduled.
+
+There are two primary components involved in DAG file processing.  The ``DagFileProcessorManager`` is an infinite loop that determines which files need
```

Comment:
I think my only comment here is the Manager also runs in a separate process to the scheduler.

(Sorry for the delay in reviewing this @john-jac!)

Reply:
No worries @ashb !  I've updated the doc with that distinction.